reframe/utility/__init__.py

```diff
@@ -565,10 +565,12 @@ def find_modules(substr, environ_mapping=None):
 
     .. code:: python
 
-       @rfm.parameterized_test(*find_modules('netcdf'))
+       @rfm.simple_test
        class MyTest(rfm.RegressionTest):
-           def __init__(self, s, e, m):
-               self.descr = f'{s}, {e}, {m}'
```

Comment:
I am not sure if you remove the description by mistake here?

Reply:
I did it no purpose. It's not needed for what we need to show.